src/tools/tidy/src/main.rs

```diff
@@ -71,6 +71,11 @@ fn filter_dirs(path: &Path) -> bool {
     skip.iter().any(|p| path.ends_with(p))
 }
 
+fn walk_many(paths: &Vec<&Path>, skip: &mut FnMut(&Path) -> bool, f: &mut FnMut(&Path)) {
```

Comment:
Interesting, seems that `&[&Path]` works with `&[&path.join ...`. I've changed it to use that notation.